src/CLI/clitree/cli-xml/interface.xml

```diff
@@ -153,11 +202,151 @@ limitations under the License.
                 ptype="RANGE_MTU" />
 	    <ACTION>python $SONIC_CLI_ROOT/sonic-cli-if.py patch_openconfig_interfaces_interfaces_interface_config_mtu ${iface} ${mtu} </ACTION>
         </COMMAND>
-	<COMMAND
+		<COMMAND
             name="no mtu"
             help="Remove MTU">
             <ACTION>python $SONIC_CLI_ROOT/sonic-cli-if.py patch_openconfig_interfaces_interfaces_interface_config_mtu ${iface} 9100 </ACTION>
         </COMMAND>
-    </VIEW>
+        <COMMAND 
+            name="shutdown"
+            help="Disable the interface">
+	    <ACTION>python $SONIC_CLI_ROOT/sonic-cli-if.py patch_openconfig_interfaces_interfaces_interface_config_enabled ${iface} False </ACTION>
+        </COMMAND>
+        <COMMAND 
+            name="no shutdown"
+            help="Enable the interface">
+	    <ACTION>python $SONIC_CLI_ROOT/sonic-cli-if.py patch_openconfig_interfaces_interfaces_interface_config_enabled ${iface} True </ACTION>
+        </COMMAND>
+		<COMMAND
+			name="switchport"
+			help="Configure switchport parameters"
+			>
+			<PARAM
+				name="switchport-subcommands"
+				help="Switchport subcommands"
+				ptype="SUBCOMMAND"
+				mode="switch"
+				>
+				<PARAM
+					name="access"
+					help="Set access mode characteristics of the interface"
+					ptype="SUBCOMMAND"
+					mode="subcommand"
+					>
+					<PARAM
+						name="vlan"
+						help="Set VLAN when interface is in access mode"
+						ptype="SUBCOMMAND"
+						mode="subcommand"
+						>
+						<PARAM
+							name="vlan-id"
+							help="VLAN ID of the VLAN when this port is in access mode"
+							ptype="UINT"
+							>
+						</PARAM>
+					</PARAM>
+				</PARAM>
+
+				<PARAM
+					name="trunk"
+					help="Configure trunking parameters on an interface"
+					ptype="SUBCOMMAND"
+					mode="subcommand"
+					>
+					<PARAM
+						name="allowed"
+						help="Configure trunking parameters on an interface"
+						ptype="SUBCOMMAND"
+						mode="subcommand"
+						>
+						<PARAM
+							name="vlan"
+							help="Configure trunking parameters on an interface"
+							ptype="SUBCOMMAND"
+							mode="subcommand"
+							>
+							<PARAM
+								name="vlan-id"
+                            	help="VLAN ID of the VLAN when this port is in access mode"
+                            	ptype="UINT"
+                            	>
+                        	</PARAM>
+						</PARAM>
```

Comment:
Fix the alignment...it's hard to see the tag completions.

Reply:
Fixed